packages/common/src/utils/url-generator/dashboard.ts

```diff
@@ -51,8 +51,29 @@ export const syncedSandboxes = (teamId?: string | null) =>
 export const templates = (teamId?: string | null) =>
   appendTeamIdQueryParam(`${DASHBOARD_URL_PREFIX}/templates`, teamId);
 
-export const recent = (teamId?: string | null) =>
-  appendTeamIdQueryParam(`${DASHBOARD_URL_PREFIX}/recent`, teamId);
+export const recent = (
```

Comment:
Not sure if it makes things simpler but it might, have you considered using `new URL` and appending search params like that? I don't think we need checks for `?` or `&` in that case. [We do this for the stripe success param](https://github.com/codesandbox/codesandbox-client/pull/7248/files#diff-c18c2f44efab0a0400faf6f5240fbc37d3d69e50ecea267767e8d15e59c2070bR21-R31). This isn't a blocker but I was curious!

Reply:
I don't like how it's implemented either but because we only have the path, the URL constructor fails because it expects a full URL. :/